posthog/test/test_feature_flag.py

```diff
@@ -2035,7 +2035,7 @@ def test_cohort_cache_no_unnecessary_queries(self):
         )
         Person.objects.create(team=self.team, distinct_ids=["example_id"], properties={"email": "tim@posthog.com"})
 
-        with self.assertNumQueries(6):
+        with self.assertNumQueries(16):
```

Comment:
Very good point, this was because the previous HogQL-based approach required all `Filter`'s to have `team`, which for some reason causes the persons-on-events instance setting to be selected a lot. For now just rolled back the `cohort.py` change